app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentListItem.kt

```diff
@@ -225,6 +227,7 @@ sealed class OverdueAppointmentListItem : PagingItemAdapter.Item<UiEvent> {
         R.drawable.ic_overdue_call
       }
       binding.callButton.setImageResource(callButtonDrawable)
+      binding.callButton.increaseTappableArea(verticalSpace = 40.dp, horizontalSpace = 24.dp)
```

Comment:
Should the value to increase by come from the resources and different depending on the screen size? As far as I know, large screens don't require the touch delegate, right?

Reply:
Atleast in this scenario we won't be changing this value based on screen size. Basically we want the call button to be clickable for the entire height of the card. According to the designs 40dp up and down below the button should cover it. So essentially we are making the entire right side of the card tap to show the contact sheet.

But we can move these dimensions to resources and refer them from there if you think that would be a better approach.

Reply:
Instead of hardcoding or referencing the 40dp another option we can consider is getting the view height and doing the math to get top and bottom area. That way the space will fit the card height no matter what screen density it is. But I am guessing in most scenarios this should be more than enough.

Reply:
Understood. Then my only concern would be handling this correctly if the design of the card changes to become either smaller or larger.

Reply:
Hmm, I guess than we have to do the math to fit the height of the card. Alright, removing the extension for this and adding this specifically to the `OverdueAppointmentListItem`.